packages/webapp/src/containers/Animals/Inventory/useFilteredInventory.ts

```diff
@@ -32,9 +46,10 @@ export const useFilteredInventory = (inventory: AnimalInventory[]) => {
     [AnimalsFilterKeys.LOCATION]: locationsFilter,
   } = useSelector(animalsFilterSelector);
 
-  return useMemo(() => {
+  const filterMatches = useMemo(() => {
     return inventory.filter((entity) => {
       const animalOrBatchMatches =
+        (showOnlySelected && selectedInventoryIds.includes(entity.id)) ||
```

Comment:
This line is unnecessary too because it will always be false at this step (due to early return)

Reply:
yup forgot that in too when I was testing the other solution! 🙏